enterprise/migrations/0105_auto_20200727_1433.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.14 on 2020-07-27 19:33
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0104_auto_20200723_1418'),
+    ]
```

Comment:
not sure what the best practice is with migrations but given the similarity between 104 and 105 it might be cleaner to roll back 104 and regenerate the migration to reflect `default=False` for the fields rather than having two separate migrations for adding the fields and then altering them. Anyone else have opinions on this?